toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -228,7 +228,8 @@ if [[ "${mode}" == "build" ]]; then # Configure base image
         docker import "${chroot_file}" $container_img
     fi
 else
-    container_img="mcr.microsoft.com/cbl-mariner/base/core:${version}"
+    # Till we publish 3.0 image, use the image created by the build phase
+    container_img="mcr.microsoft.com/cbl-mariner/${USER}-containerized-rpmbuild:${version}"
```

Comment:
This image would not be available until it has been run for build phase. We can get rid of the if-else here, and build the same image for both modes. We would need worker_chroot.tar.gz as well, so please remove the if-else from building worker_chroot.tar.gz, thanks